### ISSUE TITLE
Long term storage of images

### ISSUE THREAD

**Issue by edickens** [FreeUKGen/MyopicVicar#492]
Is it worth storing images for ever?

**Comment by edickens** [comment]
I will soon be uploading images of all the parish registers for West Sussex.  This is 600GB.  Lemon tells me that we have 6TB of storage.  But if we were to get all the images of all the counties, we could be talking about 40TB.  But how often will they be accessed?  Is this an expensive way to save images in the long term?  And this is just FreeREG.
I know that we would like to present images of the registers to researchers, and we need them for checking reported transcription errors.  But is this possible or reasonable?  The images are most likely to be available to view elsewhere and we do not have permission to display the images to researchers for most of the images that we have.  We are to use them for transcribing only.
Also, if we correct a transcription, normally we just add the researcher's version, rather than alter the original, so we do not need the images to do this.
I think we could delete images say six months after they have been transcribed.  The initial transcription will have been checked by then.  This should be programmed in when we get to the stage of matching images with transcriptions so that we can be sure that the image has been transcribed.
This is not an urgent problem, but we need to agree the policy.

**Comment by edickens** [comment]
I have been tidying up the images that were allocated a long time ago.  There are some from 2013.  I have checked and these have not been transcribed and the transcriber seems to have stopped work.  I have therefore returned the images to the pool.  The problem now is to record the fact that we have untranscribed images in the server pool.
I would like to "Create a Batch Header", with fields to include the range of years involved, but no associated entries.  When we find a transcriber, then we can tell them to enter the records into this Batch.
E
